### PR TITLE
Snippets  to FormTypes with full class names  Sf 2.8+

### PR DESCRIPTION
**Pull request by cerobit**
I use  sffuse...typing an extra f ( of form)  to keep short sfuse completion list shot  and clean.

Right now there are ~32  symfony form types  and  its better see them only after type sffuse

### CODE REVIEW

File: UltiSnips/php_symfony_form_type.snippets [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+endsnippet
+
+snippet sffusetimezonetype "Symfony Form TimezoneType"
+use Symfony\Component\Form\Extension\Core\Type\TimezoneType;
+endsnippet
+
+snippet sffuseurltype "Symfony Form UrlType"
+use Symfony\Component\Form\Extension\Core\Type\UrlType;
+endsnippet
+
```
Comment (voronkovich) [inline UltiSnips/php_symfony_form_type.snippets]: @cerobit, remove this trailing blank line, please.

Comment (cerobit) [reply UltiSnips/php_symfony_form_type.snippets]: Ok Done !

**Comment by voronkovich** [comment]
Merged! Thanks for the contribution, @cerobit!

**Comment by voronkovich** [comment]
@cerobit, BTW you can use this cool vim plugin to add use statements quickly: https://github.com/arnaud-lb/vim-php-namespace (but it requires ctags).

**Comment by cerobit** [comment]
Thanks to you for your awesome work..@voronkovich, about namespaces   i'm using  vim-php-namespace  and  vim-gutentags  but  vim-guttentags its not parsing vendor/  ....   so vim-php-namespace wont complete vendor classes.

Talking about awesome plugins, i'm really pleased with https://github.com/phpvim/phpcd.vim  its amazing blazing fast php code completion (using Php reflection Mechanism).

**Comment by voronkovich** [comment]
@cerobit, for updating tags it's better to use tools that listen file system events like Guard. I use this: https://github.com/everzet/guard-ctags-composer and https://github.com/universal-ctags/ctags

**Pull request merged**